sdk/include/xdk/setypes.h

```diff
@@ -675,6 +675,10 @@ typedef struct _SID_AND_ATTRIBUTES_HASH {
 #define NETWORKSERVICE_LUID  {0x3e4, 0x0}
 #define IUSER_LUID           {0x3e3, 0x0}
 
+/* Logon session reference flags */
+
+#define LOGON_SESSION_MARK_FOR_EVENTUAL_TERMINATION   0x0001
```

Comment:
```suggestion
#define SEP_LOGON_SESSION_TERMINATION_NOTIFY    0x0001
```
Calling the function you wrote is the act of "marking" for termination by setting the flag, but the flag name indicates the action that needs to be done.